content/guides/comparison.md

```diff
@@ -10,6 +10,8 @@ contributors:
 
 webpack is not the only module bundler out there. If you are choosing between using webpack or any of the bundlers below, here is a feature-by-feature comparison on how webpack fares against the current competition.
 
+<div class="scroll-table">
```

Comment:
Does it change anything to apply it to all? I'm not saying this is necessarily the best approach, it just seems like applying the same styles everywhere wouldn't change or hurt the styling of the other tables.